Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivateForTesting.h

```diff
@@ -97,6 +97,8 @@ struct WKAppPrivacyReportTestingData {
 - (void)_didDismissContactPicker;
 - (void)_dismissContactPickerWithContacts:(NSArray *)contacts;
 
+- (void)_renderTreeExternalRepresentation:(void(^)(NSString *))callback WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
I wonder if this should have the same naming pattern as the one that returns a string rep (or web archive or main resource data?)